FrejaEidClient/src/main/java/com/verisec/frejaeid/client/beans/general/OrganisationIdAttribute.java

```diff
@@ -0,0 +1,74 @@
+package com.verisec.frejaeid.client.beans.general;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+
+public class OrganisationIdAttribute {
+
+    private final String key;
+    private final String friendlyName;
```

Comment:
**[MEDIUM]**
I've named it displayText on the backend.

Reply:
Done